### PR TITLE
ci: add namespace support required for disruptive pods MQ-325

### PR DESCRIPTION
**Pull request by blaisedias**
+  a bunch of other changes to remove hardcoded values.

### CODE REVIEW

**Comment by blaisedias** [comment]
bors try

**Review by chriswldenyer — APPROVED** [review]
LGTM

File: test/e2e/common/util.go [hunk 1]
```diff
@@ -19,13 +19,24 @@ import (
 	. "github.com/onsi/gomega"
 )
 
+const NSE2EPrefix = "e2e-maya"
+const NSDefault = "default"
 const NSMayastor = "mayastor"
 const CSIProvisioner = "io.openebs.csi-mayastor"
+const DefaultVolumeSize = 64
```
Comment (paulyoong) [inline test/e2e/common/util.go:26]: Nit: Might be nice to include the units in the name i.e. DefaultVolumeSizeMb

File: test/e2e/common/util_cleanup.go [hunk 2]
```diff
@@ -333,28 +362,38 @@ func CleanUp() bool {
...
 				logf.Log.Info("Deleting", "storageClass", sc.Name)
 				_ = gTestEnv.KubeInt.StorageV1().StorageClasses().Delete(context.TODO(), sc.Name, metav1.DeleteOptions{GracePeriodSeconds: &ZeroInt64})
 			}
 		}
 	} else {
-		logf.Log.Info("Error listing storage classes", "error", delScErr)
+		errs = append(errs, err)
+	}
+
+	//tidy up namespaces?
```
Comment (paulyoong) [inline test/e2e/common/util_cleanup.go:382]: Should this comment be removed? You are removing the namespaces.

File: test/e2e/common/util_mayastor_crds.go [hunk 3]
```diff
@@ -297,10 +297,7 @@ func GetNexusState(uuid string) (string, error) {
 // A volume is published if the "targetNodes" field exists in the MSV.
 func IsVolumePublished(uuid string) bool {
 	_, err := getMsvFieldValue(uuid, "status", "targetNodes")
-	if err != nil {
-		return false
-	}
-	return true
+	return err != nil
```
Comment (paulyoong) [inline test/e2e/common/util_mayastor_crds.go:300]: I think you have inverted the logic here.

Comment (blaisedias) [reply test/e2e/common/util_mayastor_crds.go:300]: Good spot! fixed.

File: test/e2e/common/util_cleanup.go (hunk 2 above)
Comment (blaisedias) [reply test/e2e/common/util_cleanup.go:382]: Done.

**Review by paulyoong — APPROVED** [review]

File: test/e2e/common/util.go (hunk 1 above)
Comment (blaisedias) [reply test/e2e/common/util.go:26]: Will do, I spotted another hard coded setting in RunFio, will fix both.

Comment (blaisedias) [reply test/e2e/common/util.go]: Fixed now

**Comment by blaisedias** [comment]
bors try

**Comment by blaisedias** [comment]
bors merge

**Pull request merged**